static/templates/settings/organization_permissions_admin.hbs

```diff
@@ -125,6 +125,70 @@
+                        {{#each msg_edit_limit_dropdown_values}}
+                        <option value="{{0}}">{{1.text}}</option>
+                        {{/each}}
+                    </select>
+                    <div class="dependent-inline-block">
+                        <label for="id_realm_message_content_edit_limit_minutes" class="inline-block realm-time-limit-label">
+                            {{t 'N' }}:
+                        </label>
+                        <input type="text" id="id_realm_message_content_edit_limit_minutes"
+                          name="realm_message_content_edit_limit_minutes"
+                          class="admin-realm-time-limit-input prop-element"
+                          autocomplete="off"
+                          value="{{ realm_message_content_edit_limit_minutes }}"/>
+                    </div>
+                </div>
+
+                {{> settings_checkbox
+                  setting_name="realm_allow_community_topic_editing"
+                  prefix="id_"
+                  is_checked=realm_allow_community_topic_editing
+                  label=admin_settings_label.realm_allow_community_topic_editing}}
+
+                {{> settings_checkbox
+                  setting_name="realm_allow_edit_history"
+                  prefix="id_"
+                  is_checked=realm_allow_edit_history
+                  label=admin_settings_label.realm_allow_edit_history}}
+
+                <div class="input-group">
+                    <label for="realm_msg_delete_limit_setting" class="dropdown-title">
+                        {{t "Allow message deleting" }}
+                        <i class="fa fa-info-circle settings-info-icon realm_allow_message_deleting_tooltip tippy-zulip-tooltip"
+                          aria-hidden="true" data-tippy-content="{{t 'Administrators can always delete any message.' }}"></i>
+                    </label>
+                    <select name="realm_msg_delete_limit_setting" id="id_realm_msg_delete_limit_setting" class="prop-element">
+                        {{#each msg_delete_limit_dropdown_values}}
+                        <option value="{{0}}">{{1.text}}</option>
+                        {{/each}}
+                    </select>
+                    <div class="dependent-inline-block">
+                        <label for="id_realm_message_content_delete_limit_minutes" class="inline-block realm-time-limit-label">
+                            {{t 'N' }}:
+                        </label>
+                        <input type="text" id="id_realm_message_content_delete_limit_minutes"
+                          name="realm_message_content_delete_limit_minutes"
+                          class="admin-realm-time-limit-input prop-element"
+                          autocomplete="off"
+                          value="{{ realm_message_content_delete_limit_minutes }}"/>
+                    </div>
+                </div>
+            </div>
+        </div>
+
```

Comment:
Can you update the docs for how to change these settings to make sure they point to the new location in these first two commits?

Reply:
Updated the docs.